docs/_blog/bdk_rn_making_of.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "`bdk-rn`: How bdk-rn was developed"
+description: "bdk-rn: React Native version of BitcoinDevKit.  Insight into how bdk-rn was developed"
+authors:
+    - Bitcoin Zavior
+date: "2022-07-10"
+tags: ["BDK-RN","Development","Architecture"]
+hidden: true
+draft: false
+---
+The **BitcoinDevkit**'s **React Native** version (`bdk-rn`) makes it easy to develop bitcoin applications for both Android and IOS mobile platforms. Using `bdk-rn` knowledge of the underlying bitcoin and bdk api is not required and using `bdk-rn` is similar to using any other RN module. The goal is **Rapid Bitcoin Application Development** by doing the heavy lifting in advance and providing a reusable package for other developers to use. Developers simply install using using `yarn add` and start using it in a React Native Project. The native code, rust lang implementation, confugurations any other details are all taken care of by `bdk-rn`.
+
+This article is **NOT a guide on how to use bdk-rn** to build a bitcoin Application, rather this is an insight into how `bdk-rn` was developed. For help on how to use `bdk-rn` to develop a bitcoin Wallet or Application please refer to the user guide in the readme on github: https://github.com/LtbLightning/bdk-rn#usage. There will be `how to guides` published shortly on getting started with `bdk-rn`.
```

Comment:
I think this link (https://github.com/LtbLightning/bdk-rn#usage) should be made a direct link inside a few words preceding, for example like so:

please refer to the [user guide in the readme](https://github.com/LtbLightning/bdk-rn#usage) on github

Reply:
Done ✅